livre/core/util/FrameUtils.cpp

```diff
@@ -58,7 +58,7 @@ uint32_t FrameUtils::getNext( uint32_t current, const int32_t delta ) const
     if( _frameRange == INVALID_FRAME_RANGE )
         return INVALID_FRAME;
 
-    const uint32_t interval = std::abs( _frameRange[1] - _frameRange[0] );
+    const uint32_t interval = _frameRange[1] - _frameRange[0];
```

Comment:
(_frameRange[1] - _frameRange[0]) can never give a negative result, as frameRange is unsigned. The animation can still go on the other direction and this would still work, afaik